octoprint_mrbeam/static/js/unref.js

```diff
@@ -143,5 +143,32 @@ Snap.plugin(function (Snap, Element, Paper, global) {
 			return src_elem;
 		}
 	};
+	
+	/**
+	 * dereferences all class="..." attributes and replaces them with style="..."
+	 *
+	 * @returns {undefined}
+	 */
+	Element.prototype.unref_classes = function (classDefs = null) {
```

Comment:
Is this method used anywhere? Looks like it's just a stub without functionality?